zrml/prediction-markets/src/migrations.rs

```diff
@@ -417,8 +411,8 @@ use frame_support::dispatch::EncodeLike;
 use sp_runtime::SaturatedConversion;
 use zeitgeist_primitives::types::{MarketDispute, OldMarketDispute};
 
-const PREDICTION_MARKETS_REQUIRED_STORAGE_VERSION: u16 = 7;
-const PREDICTION_MARKETS_NEXT_STORAGE_VERSION: u16 = 8;
+const PREDICTION_MARKETS_REQUIRED_STORAGE_VERSION: u16 = 6;
+const PREDICTION_MARKETS_NEXT_STORAGE_VERSION: u16 = 7;
```

Comment:
Interesting, it seems the storage version was not bumped in the migration of [ZIP-0 PART I](https://github.com/zeitgeistpm/zeitgeist/pull/862/files#diff-7eef70e0e24e29abbdd8ae4c5949b3d7633f1fa7b844e9b0e7ff3227cc344865) that's why the storage version on-chain is 6 but the storage version in the code is 7. Can you confirm this?

Reply:
Oh I see the problem here. Look at this https://github.com/zeitgeistpm/zeitgeist/blob/d328b2f929c65ee92e95bf9eb629b4be0cc5e9ec/zrml/prediction-markets/src/migrations.rs

The last migration wasn't for the prediction markets pallet, but for the authorized pallet. I accidentally bumped the storage version of the prediction markets pallet, because I did a previous migration in that PR for the prediction markets pallet, but forgot to reduce the storage version again.

Is there any problem that we have `const STORAGE_VERSION: StorageVersion = StorageVersion::new(7);` since the last release? So, I mean we noted the storage version hard coded, but the on-chain storage still contains the old version. So I think we can just leave it as it is now. So the next Storage version step is from 6 to 7.

Reply:
It would introduce problems when another project started to use the prediction market pallet with storage version 7, as the current migration would require the version 6 and thus it would not execute in the context of the other project. However, since PM is a very specialized pallet and I am not aware of any other projects using it, this should be fine.

Authorized has the correct storage version, doesn't it?

Reply:
I like that you always think ahead of time. I see this in your forward thinking when it comes to management and maintenance for the project. This is a feature of lead developers.

Yeah in regards to authorized we have version 3 currently on main-net, which was bumped by [this](https://github.com/zeitgeistpm/zeitgeist/blob/d328b2f929c65ee92e95bf9eb629b4be0cc5e9ec/zrml/prediction-markets/src/migrations.rs#L434-L435). 

We can see 3 here too, so yeah, should everything be alright:

https://github.com/zeitgeistpm/zeitgeist/blob/d328b2f929c65ee92e95bf9eb629b4be0cc5e9ec/zrml/authorized/src/lib.rs#L57